cdap-support-bundle/src/main/java/io/cdap/cdap/support/lib/SupportBundleRequestFileList.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.support.lib;
+
+import java.util.List;
+
+public class SupportBundleRequestFileList {
```

Comment:
Missing javadoc. Why need this class?

Reply:
Because that's the format when customer request specific files to download
Request: curl -X POST "http://localhost:11011/api/v3/support/bundles/236c75c6-baf1-46ff-9519-2d88aff77566/download" -d '{“request”:"files":["testpipeline/82b21b3e-11c6-11ec-81af-0000009bb312.json"]}}'
Response: ZIP file binary stream with file name header “bundle-[current_time_millis].zip”

Reply:
But why the request need to be `{"request": {"files":[...]}}` instead of just a request as a object `{"files":[], "whateverotherfields":...}`?